.travis.yml

```diff
@@ -19,7 +19,7 @@ services:
 
 env:
   global:
-  - DOCKER_VERSION=1.11.1-0~trusty
+  - DOCKER_VERSION=1.12.0-0~trusty
```

Comment:
I cannot comment on lines below, but I don't think you need lines 27-28, because Travis already comes with docker installed (in fact the attempt to reinstall was failing for me in other PRs). We only need to install docker-compose, and only because we want a later version.